.pipelines/cherrypick/CherryPick.yml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+name: Cherry Pick commits from PR
+
+trigger:
+- fasttrack/*
+- tvuong/fasttrack/*
+pr: none
+
+variables:
+  - name: pipelinesRepoFolder
+    value: 'CBL-Mariner-Pipelines'
+
+  - name: pipelinesRepoRoot
+    value: $(Agent.BuildDirectory)/$(pipelinesRepoFolder)
+
+resources:
+  repositories:
+    - repository: CBL-Mariner-Pipelines
+      type: git
+      name: mariner/CBL-Mariner-Pipelines
+
+jobs:
+  - job: 'CherryPick'
+    pool: mariner-core-x64-1es-ubuntu-22.04
```

Comment:
Do we need this specific pool? Or would a generic `linux` be OK?

If we do need this specific agent, please use the `Agent pools (DEV)` group and the `$(DEV_AMD64_Managed)` pool inside it. This way, if the name ever changes, we won't break this pipeline.

Reply:
Oh I was not aware of this. I was using the same agent pool used by autopatching/autoupgrade tools. I'm not sure if generic linux would work, since the script that installs dependencies relies on the OS being ubuntu. That can change in the future though, so we can revisit this later.

Reply:
No problem, it's a fairly new variable group. And if you're saying that Ubuntu is what we need, then use `$(DEV_AMD64_Ubuntu_Managed)`.